infra/bracin/bin/update-nixpkgs

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+branch=$(jq -r '.branch' $DIR/../nix/src.json)
+repo=$(jq -r '.repo' $DIR/../nix/src.json)
+owner=$(jq -r '.owner' $DIR/../nix/src.json)
+
+commit=$(curl --silent \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/$owner/$repo/branches/$branch \
+         | jq -r .commit.sha)
+
+archive_sha=$(nix-prefetch-url \
+                 https://github.com/$owner/$repo/archive/$commit.tar.gz \
+                 --unpack \
+              2>/dev/null)
+
+jq -n \
+   --arg branch $branch \
+   --arg repo $repo \
+   --arg owner $owner \
+   --arg rev $commit \
+   --arg sha256 $archive_sha \
+   '{$owner, $repo, $branch, $rev, $sha256}' \
+   > $DIR/../nix/src.json
```

Comment:
Two questions here, assuming src.json is only going to be used by this project?

- why not put it under infra/bracin?
- if it stays under ../nix/ maybe give it a name that indicates it's used by infra/bracin?